en/intro.rst

```diff
@@ -272,7 +272,10 @@ To address this issue, CUBRID introduces the new concept of the Click Counter th
     *   **MULTISET**: A collection type that allows the duplication of elements. The order of entry is not considered.
     *   **LIST**: A collection type that allows the duplication of elements. Unlike with **SET** and **MULTISET**, the order of entry is maintained.
 
+*   **JSON**
+
+    JavaScript Object Notation (JSON) has become the de facto standard for data-interchanging. JSON, one of the semi-structured data, is not allowed to reside in the relational data model. In CUBRID, However, you can create and query JSON documents using :ref:`SQL functions for JSON<json-fn>`. and you can define a :ref:`JSON data type<json-data-type>` column and stores a JSON document into the column.
```

Comment:
1.In CUBRID, However, you can create and query JSON documents using :ref:`SQL functions for JSON<json-fn>`.and you can define...

- In CUBRID, However, -->  In CUBRID, however, ('However' change to lower case)

- using :ref:`SQL functions for JSON<json-fn>`. and you can define -->  using :ref:`SQL functions for JSON<json-fn>`. You can... (delete 'and')

Reply:
I'd like to suggest changes as @charischau .

Reply:
I fixed, thank you.